### DISCUSSION TITLE
About pop-up fix

### DISCUSSION THREAD

**Discussion by SylvainDx** [Clooos/Bubble-Card#970] [Questions about config, custom styles and templates] [answered]
Hello,

I'd like to activate the pop-up fix with the modification of the configuration.yaml

quick question : is th path still good ?
  - /hacsfiles/Bubble-Card/bubble-pop-up-fix.js

If I look my file structure, from the configuration.yaml, the HACS files are in ./www/community/Bubble-Card/

**Comment by MrBearPresident** [comment, marked as answer]
Yes this should still be good. see: https://github.com/Clooos/Bubble-Card?tab=readme-ov-file#pop-up-initialization-fix

You could also use `/local/community/Bubble-Card/bubble-pop-up-fix.js`
